<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Alert</class>
 <widget class="QDialog" name="Alert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lbl_text">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_alert_icon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>62</width>
     <height>52</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gb_button">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
